### PR TITLE
chaos: added etcd disruption junit testcase for Sippy

### PR DESCRIPTION
**Pull request by tsebastiani**

### CODE REVIEW

File: ci-operator/step-registry/redhat-chaos/pod-scenarios/etcd-disruption/redhat-chaos-pod-scenarios-etcd-disruption-workflow.yaml [hunk 1]
```diff
@@ -28,5 +28,6 @@ workflow:
       TELEMETRY_LOGS_BACKUP: "True"
       TELEMTRY_CLI_PATH: "/usr/local/bin/oc"
       TELEMETRY_EVENTS_BACKUP: "True"
+      JUNIT_TESTCASE: "[Jira:"Etcd"] etcd pod disruption"
```
[the remark this answers is not in the document]
Comment (chaitanyaenr) [reply ci-operator/step-registry/redhat-chaos/pod-scenarios/etcd-disruption/redhat-chaos-pod-scenarios-etcd-disruption-workflow.yaml:31]: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/55530/pull-ci-openshift-release-master-ci-operator-config/1823374529171296256

**Comment by tsebastiani** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.17-nightly-krkn-hub-tests

**Review by chaitanyaenr — APPROVED** [review]
/lgtm

**Pull request merged**